src/ec/packages/ec-component-timeline/timeline.story.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable no-param-reassign */
+
+import { storiesOf } from '@storybook/html';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+import iconPath from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+
+import demoData from './demo/data';
+
+import timeline from './timeline.html.twig';
+import notes from './docs/timeline.md';
+
+storiesOf('Components/Timeline', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withCode)
+  .addDecorator(withNotes)
+  .add(
+    'default',
+    () => {
+      const fullDemoData = {
+        ...demoData,
+        icon_path: iconPath,
+        hide: {
```

Comment:
`hide` could (should?) come from `demoData`, not from the story
it might also be interesting to add a knob on the `hide.from` and `hide.to` values 😉